packages/compare/src/compare.ts

```diff
@@ -0,0 +1,278 @@
+/**
```

Comment:
I was referring to this file @echarles (duplicate of `src/compare.ts`)

Reply:
ok, the ts one. Yep, we prolly should migrate all ts code under the module packages. An have lerna to make all module happy. I will open an issue for that and we can implement in another PR.

Reply:
Sounds good.